### ISSUE TITLE
I would appreciate it if you could provide the code for pretraining

### ISSUE THREAD

**Issue by jialancong** [SilvioGiancola/ShapeCompletion3DTracking#3]
Did you only use the completion loss when pretrained on shapenet ? I am new to pytorch and shapenet....I was wondering if you could provide the code for pretraining ?

**Comment by zarzarj** [comment]
We found the pretraining not to improve performance significantly, but the pretraining code has been added in the subfolder "pretraining".
